### PR TITLE
Add the ability to send names when using the relay group API

### PR DESCRIPTION
**Pull request by lhaskins**
This PR covers adding the commands for managing relay groups and their associated tests.

### CODE REVIEW

File: lib/cog_api/fake/relay_groups.ex [hunk 1]
```diff
@@ -28,6 +32,14 @@ defmodule CogApi.Fake.RelayGroups do
   end
 
   def delete(_, %Endpoint{token: nil}), do: Endpoint.invalid_endpoint
+  def delete(%{name: name}, %Endpoint{}) do
+    relay_group = Server.show_by_key(RelayGroup, :name, name)
+    if relay_group do
```
Comment (drapergeek) [inline lib/cog_api/fake/relay_groups.ex]: This conditional seems to duplicate the functionality of the other `delete` function. Could we pull out a shared method between the two that just takes in a `relay_group`?

File: lib/cog_api/fake/relay_groups.ex [hunk 2]
```diff
@@ -38,6 +50,17 @@ defmodule CogApi.Fake.RelayGroups do
   end
 
   def add_relay(_, _, %Endpoint{token: nil}), do: Endpoint.invalid_endpoint
+  def add_relay(%{name: name}, %{relay: relay_name}, %Endpoint{token: _}) do
+    relay = Server.show_by_key(Relay, :name, relay_name)
```
Comment (drapergeek) [inline lib/cog_api/fake/relay_groups.ex]: As with the delete, this seems to be pretty similar. Is it possible to pull out the duplication into another function that is shared?

Comment (lhaskins) [inline lib/cog_api/fake/relay_groups.ex]: @drapergeek I did a bit more DRY around the add and remove relays bit than you mentioned.

**Comment by drapergeek** [comment]
LGTM

**Pull request merged**